<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPlainTextEdit" name="textReceived">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>260</y>
      <width>451</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="textSended">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>send</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>270</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>receive</string>
    </property>
   </widget>
   <widget class="QLabel" name="statusBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>460</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWrite">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>320</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>write Meta TEDS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWritePhy">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>350</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>write PHY TEDS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWriteTs">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>write TS TEDS</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonWriteTeds">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>410</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>write TS TEDS</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>811</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_TEDS">
    <property name="title">
     <string>功能</string>
    </property>
    <addaction name="importMetaTedsAction"/>
    <addaction name="importPhyTedsAction"/>
    <addaction name="importTransducerTedsAction"/>
   </widget>
   <addaction name="menu_TEDS"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="importMetaTedsAction">
   <property name="text">
    <string>读取Meta-TEDS</string>
   </property>
  </action>
  <action name="importPhyTedsAction">
   <property name="text">
    <string>读取Phy-TEDS</string>
   </property>
  </action>
  <action name="importTransducerTedsAction">
   <property name="text">
    <string>读取Transducer-TEDS</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>importMetaTedsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importMetaTedsAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importPhyTedsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importPhyTedsAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>importTransducerTedsAction</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>importTransducerTedsAction()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>405</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonWrite</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonWriteData()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>94</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>978</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonWritePhy</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonWritePhy()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>394</y>
    </hint>
    <hint type="destinationlabel">
     <x>314</x>
     <y>545</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonWriteTs</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonWriteTs()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>191</x>
     <y>431</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>554</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonWriteTeds</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>buttonWriteTeds()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>615</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>importPhyTedsAction()</slot>
  <slot>importMetaTedsAction()</slot>
  <slot>importTransducerTedsAction()</slot>
  <slot>buttonWriteData()</slot>
  <slot>buttonWritePhy()</slot>
  <slot>buttonWriteTs()</slot>
  <slot>buttonWriteTeds()</slot>
 </slots>
</ui>
